src/clusterfuzz/stacktraces/constants.py

```diff
@@ -469,6 +469,7 @@
     r'^std::sys_common::backtrace',
     r'^__rust_start_panic',
     r'^__scrt_common_main_seh',
+    r'^libgcc_s.so.1',
```

Comment:
Can you add a test? see https://github.com/google/clusterfuzz/commit/c8ddbe20fc4706cdfc8f46b79076fd7ad6b1cd9c for an example.

Reply:
Done.